Readme.adoc

```diff
@@ -120,33 +125,30 @@ If you want to use multiple filters at once, it's possible to separate them with
 
 It's also possible to create a job dedicated to check skip messages.
 
+.Job to skip example (from: link:fail-safe-check-job-should-success.yaml[fail-safe-check-job-should-success.yaml])
 [source,yaml]
 ----
 [...]
 jobs:
-  check_skip:
+  init:
     runs-on: ubuntu-latest
     outputs:
-      skip: ${{ steps.result_step.outputs.result }}
+      skip: ${{ steps.ci-skip-step.outputs.ci-skip }}
+      skip-not: ${{ steps.ci-skip-step.outputs.ci-skip-not }}
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - uses: mstachniuk/ci-skip@master
-        with:
-          commit-filter: '[skip ci];[ci skip];[skip github]'
-          commit-filter-separator: ';'
-      - run: echo "::set-output name=result::${CI_SKIP}"
```

Comment:
@GuillaumeFavelier WDYT? I moved setting outputs to action and now it's smaller. Thank you for presenting me with how the outputs work.

Reply:
Integrating the outputs directly in the action is a great idea IMO, nice one :)

Reply:
Great! Merged. Thanks for the idea of outputs!